iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -235,17 +213,55 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
     :return: a dictionary of labels to set on the bundle
     :rtype: dict
     """
-    package_name, labels = _apply_package_name_suffix(metadata_path, organization)
-
     try:
         operator_manifest = OperatorManifest.from_directory(manifests_path)
     except (ruamel.yaml.YAMLError, ruamel.yaml.constructor.DuplicateKeyError) as e:
         error = f'The Operator Manifest is not in a valid YAML format: {e}'
         log.exception(error)
         raise IIBError(error)
 
-    found_pullspecs = set()
-    operator_csvs = []
+    conf = get_worker_config()
+    organization_customizations = conf['iib_organization_customizations'].get(organization, [])
+
+    annotations_yaml = _get_package_annotations(metadata_path)
+    package_name = annotations_yaml['annotations'][
+        'operators.operatorframework.io.bundle.package.v1'
+    ]
+    labels = {}
+    if organization_customizations:
+        # Perform the customizations in order
+        for customization in organization_customizations:
+            customization_type = customization['type']
+            if customization_type == 'package_name_suffix':
+                package_name, labels = _apply_package_name_suffix(metadata_path, organization)
```

Comment:
Could you refactor `_apply_package_name_suffix`, `_get_bundle_metadata`, and `_adjust_csv_annotations` to accept `customization` rather than `organization`? Then each private function doesn't need to be aware of the `iib_organization_customizations` configuration and I think that'd be cleaner.

Reply:
Yes, this is on my radar. I plan to make this change in my next PR when I introduce a new customization. Would it be okay if I did it in my next PR?

Reply:
I'd rather do it now, but I won't block my approval on it.